pystruct/learners/ssvm.py

```diff
@@ -18,6 +23,15 @@ def __init__(self, model, max_iter=100, C=1.0, verbose=0,
         self.n_jobs = n_jobs
         self.logger = logger
 
+        if self.n_jobs == -1:
+            self._n_jobs = cpu_count()
```

Comment:
yes. The reason is that when you do GridSearchCV for example, new instances will be created using an sklearn internal ``clone`` based on the init parameters. So if you do it this way, the estimators for the different grid points will share a pool. And if someone sets the pool size as one of the options in the grid search, it will just break.